lib/agent/plugins/control-panel/interval/test/interval_spec.js

```diff
@@ -0,0 +1,193 @@
+var assert = require("assert"),
+    sinon  = require('sinon'),
+    should = require('should'),
+    api = require('../../api'),
+    interval = require(".."),
+    bus = require("../../bus"),
+    hooks = require('../../../../hooks'),
+    common = require('../../../../../common'),
```

Comment:
Do we really need to require the original common module. Why not just build a fake `common_obj` for the purposes of the test?